### DISCUSSION TITLE
How to access Orchard properties in Razor within a shape created by New.MyShape?

### DISCUSSION THREAD

**Discussion by xperiandri** [OrchardCMS/OrchardCore#9349] [Q&A]
I create a shape like `@await DisplayAsync(await New.CarModelItem(CarModel: model, Culture: currentCulture))`
What properties are available to me in the shape cshtml?
What type should I annotate my shape with?

**Comment by jtkech** [comment]
If you don't specify a `@model` directive in your view, so using a dynamic one

`Model.CarModel`

That you can cast to the type of `CardModel`

**Reply by xperiandri** [reply]
Yep, but can I access Orchard or MVC stuff from it? Like `Culture`, etc.?
